File: palindrome.js

const checkIfPalindrome = (word) => {
    //A palindrome is a word, number, phrase, or other sequence of symbols that reads the same backwards as forwards, such as madam or racecar
    let reverseWord;
    reverseWord = word.split("").reverse().join("");
    // check if its a palindrome
    if (word == reverseWord) {
        alert(`${word} is a palindrome`)
    } else {
        alert(`${word} is not a palindrome ${reverseWord}`)
       return reverseWord;
    }
}

const input = prompt("Enter a word: ");
checkIfPalindrome(input);

File: palindrome.py
def check_palindrome(word):
    if word == word[::-1]:  # Check if the word is a palindrome
        return True, word
    else:
        return False, word[::-1]

input = input("Enter a word: ")
is_palindrome, result = check_palindrome(input)

if is_palindrome:
    print(f"{input} is a palindrome.")
else:
    print(f"{input} is not a palindrome. Reversed: {result}")
